packages/amplication-client/package.json

```diff
@@ -17,9 +17,7 @@
     "events": "3.3.0",
     "formik": "2.2.9",
     "graphql": "15.0.0",
-    "lodash.debounce": "4.0.8",
-    "lodash.keyby": "4.6.0",
-    "lodash.omit": "4.5.0",
+    "lodash": "^4.17.21",
```

Comment:
Please use exact version because its a service and not a lib, please check the --save-exact npm's flag